articles/house-price-prediction/index.md

```diff
@@ -0,0 +1,138 @@
+<!--more-->
+### Prerequisites
+There are a few theoretical and programming pre-requisites required for this article. They are helpful but not required to understand the article.
+
+1. [Introduction to Pandas](/engineering-education/data-analytics-using-pandas/)
+2. [Introduction to Supervised Learning Algorithms using Scikit-Learn](/engineering-education/supervised-learning-algorithms/)
+
+### Introduction
+The problem falls under the category of supervised learning algorithms. The dataset we'll be using is the Boston Housing Dataset. The dataset comprises 13 input features and one target feature. The input features include features that may or may not impact the price.
+
+#### Dataset
+The Boston data frame has 506 rows and 14 columns. Each row comprises one data-point and contains details about a plot. Various features affect the pricing of a house. The Boston housing dataset has 14 features, out of which we will use 13 to train the model. The 14th feature is the price, which we'll use as our target variable.
+
+The table gives the list of features included in the dataset, along with their respective descriptions.
+
+Features | Description
+--- | ---
+**crim** | *per capita crime rate by town*
+**zn** | *proportion of residential land zoned for lots over 25,000 sq.ft.*
+**indus** | *proportion of non-retail business acres per town.*
+**chas** | *Charles River dummy variable (= 1 if tract bounds river; 0 otherwise).*
+**nox** | *nitrogen oxides concentration (parts per 10 million).*
+**rm** | *average number of rooms per dwelling.*
+**age** | *proportion of owner-occupied units built prior to 1940*
+**dis** | *weighted mean of distances to five Boston employment centers.*
+**tax** | *full-value property-tax rate per \$10,000.*
+**ptratio** | *pupil-teacher ratio by town.*
+**black** | *1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town.*
+**lstat** | *lower status of the population (percent).*
+**medv** | *median value of owner-occupied homes in \$1000s.*
+
+
+### Approach taken
+We'll use the Random Forest regression algorithm to predict the price of the houses. In this article, we'll consider machine learning algorithms as a black box that fits the data. 
+
+This article focuses more on the machine learning pipeline. For more information on the Random Forest algorithm, I suggest looking into this [video](https://www.youtube.com/watch?v=nxFG5xdpDto).
+
+We'll begin with loading the data. Since we are using an inbuilt dataset, we'll be calling the `load_boston` function from the `sklearn.datasets` module. The data is loaded into the `data` variable. Once the data is loaded, we separate the data and target attributes of the `data` variable. We store them in variables `data` and `target` respectively.
```

Comment:
Consider changing the preposition "with" It should be:
"We'll begin by loading the data."